region/lowernorfair/east.json

```diff
@@ -1664,7 +1664,7 @@
                   ]},
                   {"or": [
                     {"and": [
-                      "canHitBox",
+                      "canPlasmaHitbox",
```

Comment:
This room was supposed to stay as `canHitbox`, right?

Reply:
Well f me